<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Prompt_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter your information below: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Username_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username_lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Passwird_lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Login_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Register_Button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Error_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Error.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password_Error_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Error.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
